### ISSUE TITLE
get_bls_state() assumes that the date_mth is null

### ISSUE THREAD

**Issue by mhollander** [keberwein/blscrapeR#4]
I tried running this simple command:
`states <- get_bls_state(date_mth=c("January 2016", "February 2016"), seasonality=FALSE)
`
Unfortunately, no matter what I write for the dates, I only get data from Dec 2016.  It appears that the error is in get_bls_state.R - you assume that there is no date passed in rather than checking for NULL.  The fix should be one line of code, thankfully.

Thanks for writing this package it's great!

**Comment by keberwein** [comment]
You were right, one line of code. Thanks for pointing this out! I applied the fix [here](https://github.com/keberwein/blscrapeR/commit/642521242f8f4a6224f23fa597b3d2d7c58a8ed0). If you update from GitHub with devtools install everything should be resolved. I will keep this issue open until we can get the new version up to CRAN.

`devtools::install_github("keberwein/blscrapeR") `

**Comment by mhollander** [comment]
Thanks for the fix.  I'm curious - I'd like to be able to download all of
the data from a start date until an end date without having to put in all
of the months (e.g. I want to get 20 years of data at once).  Any interest
in adding this feature?

**Comment by keberwein** [comment]
That's a good idea and is totally do-able. If you want the quick solution you can generate a list of dates, format them, and pass them to the function as an argument. This is quick 'n dirty but it works if you want an immediate solution.

```
# Generate series
monthlist <- seq(as.Date("1977/1/1"), as.Date("2016-12-01"), by = "month")
months <- format(monthlist, "%B %Y")

states <- blscrapeR::get_bls_state(date_mth=months, seasonality=TRUE)
```

**Comment by keberwein** [comment]
This issue was closed with CRAN release 2.1.1 https://github.com/keberwein/blscrapeR/commit/7126678a55fdfbfccf2c18f04cbf7a23f85ee971
